src/renderer/features/governance/model/lockPeriods.ts

```diff
@@ -30,11 +25,27 @@ sample({
   target: $lockPeriods,
 });
 
+sample({
+  clock: delegateValidateModel.validate,
+  source: {
+    apis: networkModel.$apis,
+    chains: networkModel.$chains,
+  },
+  fn({ apis, chains }, { transaction }) {
+    const chain = chains[transaction.chainId];
+    const api = apis[transaction.chainId];
+
+    return {
+      api,
+      chain,
+    };
+  },
```

Comment:
Can do this, if you don't mind
```suggestion
  fn: ({ apis, chains }, { transaction }) => ({
    api: apis[transaction.chainId],
    chain: chains[transaction.chainId],
  }),
```